docs/manual/cluster-admin/basic-management-operations.md

```diff
@@ -151,4 +151,117 @@ Another example is to restart the whole cluster:
\ No newline at end of file
+You can use `exit` to leave the dev-box container, and use `sudo docker exec -it dev-box bash` to re-enter it if you desire so. If you don't need it any more, use `sudo docker stop dev-box` and `sudo docker rm dev-box` to delete the docker container.
+
+## <div id="how-to-set-up-https">How To Set Up Https</div>
+
+To configure https certificate for pylon, you need to obtain a digital certificate first, and then save the digital certificate related files in the dev-box container. In the dev-box, you can find the configuration file `services-configuration.yaml`, and then you need to Enter the file path of the saved digital certificate into the `services-configuration.yaml` file. You can choose a self-signed certificate or a certificate issued by a CA. Next, we will first demonstrate the configuration process of a self-signed certificate. The configuration process for the two types of certificates is similar.
+
+### Configure A Self-Signed Certificate
+
+
+#### 1. Enter the dev-box onctainer
+
+You need to use [`paictl`](#pai-service-management-and-paictl) to enter the container in the following way:
+
+```bash
+sudo docker exec -it dev-box bash
+```
+
+#### 2. Create a folder in the dev-box container
+When you enter the container, you need to create a folder and generate a self-signed certificate in this folder. We can create an ssl folder in the home folder.
+
+``` bash
+mkdir /home/ssl
+cd  /home/ssl
+```
+#### 3. Generate RSA private key with openssl
+The next commands you enter will use the FileName parameter multiple times. You can choose a suitable file name to replace FileName.
+
+``` bash
+openssl genrsa -des3 -out FileName.key 1024
+```
+
+In this step, password will be asked.
+
+#### 4. Generate certificate request
+
+```bash
+SUBJECT="/C=US/ST=Washington/CN=FileName"
+openssl req -new -subj $SUBJECT -key FileName.key -out FileName.csr
+```
+
+#### 5. Generate certificate
+
+```bash
+mv FileName.key FileName.origin.key
+openssl rsa -in FileName.origin.key -out FileName.key
+openssl x509 -req -days 3650 -in FileName.csr -signkey FileName.key -out FileName.crt
+```
+
+#### 6. Final result
+
+In the current directory, you will find 4 files:
+
+<div  align="center">
+<img src="./imgs/aad/openssl_result.png" alt="paictl overview picture" style="float: center; margin-right: 10px;" />
+</div>
+
+#### 7. Configure services-configuration.yaml
+
+
+If you are configuring for the first time, `services-configuration.yaml` may not exist in the dev-box container. You should follow the procedure below to change the configuration file and make it effective. Close the pylon service, pull the OpenPAI configuration file `services-configuration.yaml` to the local, change the configuration file, upload the configuration file, and restart the pylon service. The commands you need are:
```

Comment:
`If you are configuring for the first time` -> `If you are configuring PAI service for the first time`
` Close the pylon service,` -> ` Then, stop the pylon service,`